packages/cli/src/commands/pg/backups/unschedule.ts

```diff
@@ -14,7 +14,7 @@ export default class Unschedule extends Command {
   };
 
   static args = {
-    database: Args.string({description: 'config var exposed to the owning app containing the database configuration'}),
+    database: Args.string({description: 'globally unique name or ID of the database attachment. If omitted, we use an arbitrary postgres database.'}),
```

Comment:
I'd feel weary reading about my platform doing anything arbitrarily with my data lol. Would this work?

"If omitted, we use any database with a backup schedule. If there are multiple schedules, you must provide this argument."

I don't really like the start of the description either. See my DM about how we should approach this in general.

Reply:
Resolved per our discussion.